test/stress/stress_test.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package stress
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-getter"
+	"k8s.io/minikube/pkg/util"
+)
+
+const newPath = "../../out/minikube"
+const cmdTimeout = int64(90 * 1000)
+
+// TestStress runs the stress test
+func TestStress(t *testing.T) {
```

Comment:
since we are gonna have other stress tests, lets call it more meaningful name, 
in this case seems to be doing upgrade...

also consider making Sub Tests out of them
so when it will fail we know what part failed in the gopogh

For example

Stress/DownloadBinary
Stress/
Stress/UpgradeFromOld/

Reply:
done